common/changes/@microsoft/api-extractor-model/octogonz-stable-sort_2019-11-02-20-23.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor-model",
+      "comment": "Fix an issue where API reports sometimes were ordered differently depending",
```

Comment:
Unfinished sentence?

Reply:
I copy+pasted the other message into the CLI; must have gotten truncated